ebuild-writing/functions/src_install/docompress/text.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<guide self="ebuild-writing/functions/src_install/docompress/">
+<chapter>
+<title>Controllable Compression</title>
+<body>
+
+<p>
+To compress files in the destination folder <c>${D}</c>, the <c>docompress</c>
```

Comment:
I think this wrongly suggests that docompress does the actual compression and/or is required for the compression to happen. Maybe 'The docompress function can be used to control ....'

Reply:
Good point, I'll fix (although I've just move that text).